### PR TITLE
Tier2 Codeowners for framework based changes

### PR DESCRIPTION
**Pull request by jyejare**
$subject

### CODE REVIEW

**Comment by jyejare** [comment]
@JacobCallahan @ogajduse Lets gets this merged.

**Review by ogajduse — APPROVED** [review]
:heart:

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -1,3 +1,19 @@
...
+
+/pytest_fixtures/ @SatelliteQE/robottelo-tier-2-reviewers
+
+/pytest_plugins/ @SatelliteQE/robottelo-tier-2-reviewers
+
+/robottelo/ @SatelliteQE/robottelo-tier-2-reviewers
+
+/scripts/ @SatelliteQE/robottelo-tier-2-reviewers
+
+/tests/foreman/sanity/ @SatelliteQE/robottelo-tier-2-reviewers
```
Comment (Gauravtalreja1) [inline .github/CODEOWNERS:19]: wdyt?
```suggestion
/tests/foreman/sanity/ @SatelliteQE/robottelo-tier-2-reviewers

/tests/robottelo/ @SatelliteQE/robottelo-tier-2-reviewers

/docs/ @SatelliteQE/robottelo-tier-2-reviewers
```

File: .github/CODEOWNERS [hunk 2]
```diff
@@ -1,3 +1,19 @@
 # Tier Codeowners for framework based changes
 
 * @SatelliteQE/robottelo-tier-1-reviewers
+
+/* @SatelliteQE/robottelo-tier-2-reviewers
```
[the remark this answers is not in the document]
Comment (jyejare) [reply .github/CODEOWNERS:5]: * is everything opened in Robottelo irrespective of its place will be tagged for T1.

The /* is all files(not directories) on root directory to be tagged for T2.


The /* will override the first * hence all root files will be only tagged for T2.

File: .github/CODEOWNERS (hunk 1 above)
Comment (jyejare) [reply .github/CODEOWNERS:19]: I think docs are worth adding for the T2 review which I believe is being less touched last touch in that file has 2/3 years ago. We can get rid of docs if that's ACKed by all @SatelliteQE/airgun-tier-2-reviewers stakeholders. 

Rest, There is a less chance that anyone will touch `tests/robottelo` without changing anything in framework changes also tests inside `tests/robottelo` are part of CI and getting them passing is mandatory. So dont want to enforce T2 review there.

**Review by JacobCallahan — APPROVED** [review]

**Review by ColeHiggins2 — APPROVED** [review]

**Pull request merged**